cmd/authenticator/main.go

```diff
@@ -17,6 +17,8 @@ func main() {
 
 	var err error
 
+	configureLogLevel()
```

Comment:
should this be done in `config.go`?

Reply:
At first, I thought it shouldn't, as each consumer may enable debug mode however he wants to.
But then I thought it doesn't differ from any other configuration, which done through env vars.
So I will move it to there.